<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordInfo</class>
 <widget class="QWidget" name="RecordInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RecordInfo{
background-color: rgb(255, 255, 255);
}
QLabel{
font: 18pt &quot;楷体&quot;;
color: rgb(0, 0, 0);
}
QToolButton{
font: 18pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
border:2px solid rgb(0, 125, 165);
border-radius:10px;
background-color: rgb(0, 125, 165);
}

</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>1340</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
border:1px solid rgb(0,125,165);
border-radius:10px;
}</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_7">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>635</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>635</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>查询记录</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>845</x>
     <y>635</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>全部删除</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>1115</x>
     <y>635</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>635</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>导出记录</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditStart">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>635</y>
     <width>165</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>125</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;楷体&quot;;</string>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>635</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>结束
时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>635</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>开始
时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditStop">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>635</y>
     <width>165</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>125</width>
     <height>60</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;楷体&quot;;</string>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>8</month>
     <day>1</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
